system-x/services/fhir/src/main/java/software/tnb/fhir/service/Fhir.java

```diff
@@ -0,0 +1,25 @@
+package software.tnb.fhir.service;
+
+import software.tnb.common.service.Service;
+
+import java.util.Map;
+
+public abstract class Fhir implements Service {
+    private static final String FHIR_IMAGE_KEY = "fhir.image";
+    public static final String FHIR_IMAGE = "hapiproject/hapi:v4.2.0";
```

Comment:
please mirror this image to our quay organization so that we don't have to deal with dockerhub pull limits